index.html

```diff
@@ -337,17 +329,41 @@ <h3>
           can simply forward the share data to the operating system and not
           talk directly to native applications.
         </p>
-        <p>
+        <p data-link-for="ShareData">
           Mapping the <a>ShareData</a> to the share target (or operating
           system)'s native format can be tricky as some platforms will not have
           an equivalent set of fields. For example, if the target has a "text"
           field but not a "URL" field, one solution is to concatenate both the
-          <a for="ShareData">text</a> and <a for="ShareData">url</a> fields of
-          <a>ShareData</a> and pass the result in the "text" field of the
-          target.
+          <a>text</a> and <a>url</a> fields of <a>ShareData</a> and pass the
+          result in the "text" field of the target.
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The following are defined in [[!WEBIDL]]:
+      </p>
+      <ul data-sort="">
+        <li>
+          <code><dfn data-cite=
+          "!WEBIDL#exceptiondef-typeerror">TypeError</dfn></code>
```

Comment:
I wonder if we could instead link to https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror which is where `TypeError` is actually defined? WebIDL just refers to that. Maybe web specs don't normally directly link to ECMA-262. This is fine also.